commit 32d27530633334b6ca8a8d01a3f555043f6b13f9
Author: Christine Caulfield <ccaulfie@redhat.com>
Date:   Fri Jul 19 11:17:25 2013 +0100

    ccs_tool: Fix example fence device in "ccs_tool create" help
    
    Although creating fence devices using ccs_tool is not supported
    in RHEL6, the help should at least show a valid configuration.
    
    This patch is an amalgamated copy from the STABLE32 branch and
    
    Resolves: rhbz#871603
    
    Signed-off-by: Christine Caulfield <ccaulfie@redhat.com>

diff --git a/config/tools/ccs_tool/editconf.c b/config/tools/ccs_tool/editconf.c
index 7971672..34f7893 100644
--- a/config/tools/ccs_tool/editconf.c
+++ b/config/tools/ccs_tool/editconf.c
@@ -77,7 +77,7 @@ static void create_usage(const char *name)
 	  "\n"
 	  "eg:\n"
 	  "  ccs_tool create MyCluster\n"
-	  "  ccs_tool addfence apc fence_apc ipaddr=apc.domain.net user=apc password=apc\n"
+	  "  ccs_tool addfence apc fence_apc ipaddr=apc.domain.net login=apc passwd=apc\n"
 	  "  ccs_tool addnode node1 -n 1 -f apc port=1\n"
 	  "  ccs_tool addnode node2 -n 2 -f apc port=2\n"
 	  "  ccs_tool addnode node3 -n 3 -f apc port=3\n"
@@ -144,7 +144,7 @@ static void addnode_usage(const char *name)
 	fprintf(stderr, " -n --nodeid        Nodeid (required)\n");
 	fprintf(stderr, " -v --votes         Number of votes for this node (default 1)\n");
 	fprintf(stderr, " -a --altname       Alternative name/interface for multihomed hosts\n");
-	fprintf(stderr, " -f --fence_type    Type of fencing to use\n");
+	fprintf(stderr, " -f --fence_type    Name reference of fencing to use\n");
 	config_usage(1);
 	help_usage();
 
@@ -152,10 +152,10 @@ static void addnode_usage(const char *name)
 	fprintf(stderr, "Examples:\n");
 	fprintf(stderr, "\n");
 	fprintf(stderr, "Add a new node to default configuration file:\n");
-	fprintf(stderr, "  %s %s newnode1 -n 1 -f manual ipaddr=newnode\n", prog_name, name);
+	fprintf(stderr, "  %s %s newnode1 -n 1 -f wti7 port=1\n", prog_name, name);
 	fprintf(stderr, "\n");
 	fprintf(stderr, "Add a new node and dump config file to stdout rather than save it\n");
-	fprintf(stderr, "  %s %s newnode2 -n 2 -f apc -o- newnode.temp.net port=1\n", prog_name, name);
+	fprintf(stderr, "  %s %s -o- newnode2 -n 2 -f apc port=1\n", prog_name, name);
 
 	exit(0);
 }
